loca.html

```diff
@@ -3,11 +3,22 @@
 <head>
     <meta charset="utf-8"/>
     <title>KFC Maroc</title>
+    
     <link href="style.css" rel="stylesheet">
     <link rel="preconnect" href="https://fonts.googleapis.com">
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 <link href="https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&family=Rubik:ital,wght@0,300;0,400;0,500;0,600;0,700;0,900;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Viga&display=swap" rel="stylesheet">
-    </head>
+<script LANGUAGE="JavaScript">
+function change(){
+  if (window.document.formliste.liste.selectedIndex != 0)
+  window.parent.location = window.document.formliste.liste.options
+    [document.formliste.liste.selectedIndex].value
```

Comment:
Vu que tu récupère le selectedIndex 2 fois, le plus optimale est de le récupérer une fois dans au début et le stocker dans une variable
```suggestion
  let selectedIdx = window.document.formliste.liste.selectedIndex

  if (selectedIdx != 0)
    window.parent.location = window.document.formliste.liste.options[selectedIdx].value
```